src/Controller/SecurityController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Form\RegisterType;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class SecurityController extends AbstractController
+{
+    /**
+     * @Route("/inscription", name="security_registration")
+     */
+    public function registration(Request $request, EntityManagerInterface $manager, UserPasswordEncoderInterface $encoder) 
+    {
+        $user = new User();
+        $form = $this->createForm(RegisterType::class, $user);
+        $form->handleRequest($request);
```

Comment:
`$form = $this->createForm(RegisterType::class, $user)->handleRequest($request);`